app/controllers/charges_controller.rb

```diff
@@ -107,34 +107,27 @@ def validate(seller_id:, line_items:)
   def generate_description(seller_name:, item_types:)
     description = 'Thank you for supporting ' + seller_name + '.'
     if item_types.include? 'gift_card'
-      description +=
-        ' Your gift card(s) will be emailed to you when the seller opens back up.'
+      description += ' Your gift card(s) will be emailed to you when the seller opens back up.'
```

Comment:
Can you run `rubocop -a` to make sure this line isn't too long?

Reply:
Just broke it up. I'm not too sure what the best practice is for ruby for breaking up lines haha

Reply:
My own preference is to try and keep the string together so that it's easier to grep — but idc really lol

Reply:
usually I just ignore it if it's just a long string

Reply:
Only problem was that even with the previous change the line was at 82 which exceeded the default 80. Maybe we should change the default limit to 100 or so